README.md

```diff
@@ -8,6 +8,15 @@ It is available for downloading as [Starcounter.Linq](https://www.nuget.org/pack
 
 Requires Starcounter version 2.3.1 or later and .NET Framework 4.5.

```

Comment:
Does this make sense to have with the added table below?

Reply:
Agree. Removed.